pkg/controller/operators/olm/operatorgroup.go

```diff
@@ -388,6 +388,7 @@ func (a *Operator) ensureClusterRoles(op *v1alpha2.OperatorGroup) error {
 			},
 			Rules: apiEditPolicyRules,
 		}
+		//ownerutil.AddNonBlockingOwner(operatorGroupEditClusterRole, csv)
```

Comment:
I'm wondering if this would be good to have?

Reply:
Right now, this is useful for requeuing owner CSV.